NEWS

```diff
@@ -7,6 +7,9 @@ ggplot2 1.0.1
 * Improvements to order of colours and legend display for continuous colour
   scales extracted from colorbrewer palettes by `scale_*_distiller()` (@jiho, 1076)
 
+* Restore functionality for use of `..density..` in 
```

Comment:
This should go at the top of `NEWS.md` (just underneath the version number heading)